lib/verify.js

```diff
@@ -50,7 +50,7 @@ function verify (cache, opts) {
 
   return steps
     .reduce((promise, step, i) => {
-      const label = step.name || `step #${i}`
+      const label = step.name
```

Comment:
all those steps are properly named functions declared within this file, so I don't see a reason to have this fallback here